### ISSUE TITLE
Discrepancy in network options between advanced setup and settings

### ISSUE THREAD

**Issue by hippalectryon-0** [sandboxie-plus/Sandboxie#2746]
### Describe what you noticed and did

When creating a new sandbox and checking "Configure advanced options", one can set "Network access" to "Block using WFP" or "deny access".

When doing so, the configuration is *different* than when setting those exact same options in the settings panel.

**.ini with WFP from settings**:
```
AllowNetworkAccess=!<InternetAccess>,n
AllowNetworkAccess=<BlockNetAccess>,n
ClosedFilePath=<BlockNetDevices>,InternetAccessDevices
```

**.ini with WFP from new sandbox > configure advanced options**
```
ClosedFilePath=!<InternetAccess>,InternetAccessDevices
AllowNetworkAccess=<BlockNetAccess>,n
```

**.ini with denying access from settings**:
```
AllowNetworkAccess=<BlockNetAccess>,n
ClosedFilePath=!<InternetAccess>,InternetAccessDevices
ClosedFilePath=<BlockNetDevices>,InternetAccessDevices
```

**.ini with denying access from new sandbox > configure advanced options**
```
AllowNetworkAccess=!<InternetAccess>,n
```

### How often did you encounter it so far?

Everytime

### Affected program

Not relevant

### Download link

Not relevant

### Where is the program located?

Not relevant to my request.

### Expected behavior

Both methods should yield the same configuration

### What is your Windows edition and version?

Windows 10

### In which Windows account you have this problem?

Not relevant to my request.

### Please mention any installed security software

Not relevant

### What version of Sandboxie are you running?

Plus 1.8.0

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

**Comment by DavidXanatos** [comment]
already fixed: https://github.com/sandboxie-plus/Sandboxie/issues/2729

**Comment by hippalectryon-0** [comment]
This issue is _not_ fixed in 1.8.1

Here's the updated .ini:

**.ini with WFP from settings**:
```
AllowNetworkAccess=!<InternetAccess>,n
AllowNetworkAccess=<BlockNetAccess>,n
ClosedFilePath=<BlockNetDevices>,InternetAccessDevices
```

**.ini with WFP from new sandbox > configure advanced options**
```
AllowNetworkAccess=!<InternetAccess>,n
```

**.ini with denying access from settings**:
```
AllowNetworkAccess=<BlockNetAccess>,n
ClosedFilePath=!<InternetAccess>,InternetAccessDevices
ClosedFilePath=<BlockNetDevices>,InternetAccessDevices
```

**.ini with denying access from new sandbox > configure advanced options**
```
ClosedFilePath=!<InternetAccess>,InternetAccessDevices
```

**Comment by DavidXanatos** [comment]
in fact the single lines are enough to achieve what was set out to do.
At this stage of box existence there will be no entries in BlockNetAccess or BlockNetDevices groups,
only the lines which apply to all programs not in the permitted group are relevant.
Once the user opens the config and adds entries to the two Block groups the needed additional access directives will be added.

**Comment by isaak654** [comment]
I would tend to agree with the OP, the additional lines should be cleaned correctly the same moment you return to the default "untouched" network options in Sandbox Settings.

**Comment by DavidXanatos** [comment]
I will fix that for 1.8.5 or later
